packages/server/graphql/private/mutations/processScheduledLocks.ts

```diff
@@ -0,0 +1,107 @@
+import ms from 'ms'
+import sendTeamsLimitEmail from '../../../billing/helpers/sendTeamsLimitEmail'
+import getRethink from '../../../database/rethinkDriver'
+import isValid from '../../isValid'
+import {MutationResolvers} from '../resolverTypes'
+
+const processScheduledLocks: MutationResolvers['processScheduledLocks'] = async (
```

Comment:
@nickoferrall I happened to make a slightly different solution in my PR. It uses the existing scheduled job mechanism. 

The idea is to create scheduled jobs every time the limit is exceeded. It could be multiple jobs created at once - block after 30 days, alert after 7, etc.

Here is the PR: https://github.com/ParabolInc/parabol/pull/7606/files

What do you think about this solution?

Reply:
@nickoferrall I think yes. There is already a job that locks an organization. You can add sending email to it here:

https://github.com/ParabolInc/parabol/pull/7606/files#diff-0ebb6687f73dcbeef95db77a23ed552ac97457d45368062605a87c17330b9f9dR201-R207

Then you can create a reminder email job object here: 

https://github.com/ParabolInc/parabol/pull/7606/files#diff-0ebb6687f73dcbeef95db77a23ed552ac97457d45368062605a87c17330b9f9dR50

And add a job processor for it here:

https://github.com/ParabolInc/parabol/pull/7606/files#diff-88438075aa9f4efe2c41c3ee8437756a2c7a6dfd1a50e2e3718cba2660330f4bR46-R55

Reply:
Sounds good, I'll work on implementing this today 👍

Reply:
Update pushed! I've also started splitting up the `teamLimitChecks` file and changed the base of the PR to your `lock-scheduled-job` PR